maraboupy/test/test_nnet_extensions.py

```diff
@@ -0,0 +1,149 @@
+
+from maraboupy.MarabouNetworkNNet import *
+
+
+from subprocess import call
+
+property_filename = "./resources/properties/acas_property_4.txt"
+network_filename = "./resources/nnet/acasxu/ACASXU_experimental_v2a_1_9.nnet"
+
+layer = 2
+
+
+def test_write_to_file():
+    output_filename = "./maraboupy/test/ACASXU_experimental_v2a_1_9_output.nnet"
+
+    nnet_object = MarabouNetworkNNet(filename=network_filename)
+    nnet_object.writeNNet(output_filename)
+
+    call(['diff', output_filename,network_filename])
```

Comment:
Doesn't this need some sort of assert?

Reply:
I have removed this test. 

writeNNet is tested in a different test via evaluation of the two networks; here I just wanted to see what the diff looks like.